data/dictionary/encon.yaml

```diff
@@ -0,0 +1,9 @@
+word: En Con
```

Comment:
Um. EnCon, unless they followed in the footsteps of Great Con?

You might need to find someone and ask them.